azure-pipelines-conditional-integration.yml

```diff
@@ -31,6 +31,20 @@ schedules:
     - main
   always: true # run the pipeline even if there are no code changes
 
+- cron: '0 */24 * * *'
+  displayName: FeatureFlags - ForceRuntimeCodeGeneration
+  branches:
+    include:
+    - main
+  always: true # run the pipeline even if there are no code changes
+
+- cron: '0 */24 * * *'
+  displayName: FeatureFlags - ForceRuntimeCodeGeneration,UseRazorCohostServer
+  branches:
+    include:
+    - main
+  always: true # run the pipeline even if there are no code changes
+
```

Comment:
This triggers on the actual pipeline imply that these will be ignored.
![image](https://github.com/user-attachments/assets/d4430d33-e7de-442b-8241-357fc61efeab)


I suspect we need to add another cron job here for the "regular" run, and then merge this, and then turn off that override setting.

Reply:
ah interesting. It already has an existing trigger so I assumed it was used.